donate.html

```diff
@@ -312,47 +314,51 @@ <h5 class="margin-bottom-normal">{% filter ak_text:"donate_billing_information"
                       {% else %}
                       <input type="hidden" name="country" value="United States">
                       {% endif %}
-                      {% if has_shippable_products %}
-                      <div class="input-text ak-input-type-user input-field-group ak-err-below">
-                        <label for="id_address1">Billing address line 1</label>
-                        <input id="id_address1" class="" type="text" name="address1">
-                      </div>
-                      <div class="input-text ak-input-type-user input-field-group ak-err-below">
-                        <label for="id_address2">Billing address line 2 (optional)</label>
-                        <input id="id_address2" class="" type="text" name="address2">
-                      </div>
+                      {% if page.lang_or_en.iso_code == 'ja' %}
+                        {% include "./japanese_address_fields.html" %}

```

Comment:
seems like this could be needed for other countries in the future, but can just update to a switch statement or something similar as desired later